keras_cv/bounding_box/converters.py

```diff
@@ -373,6 +373,9 @@ def _image_shape(images, image_shape, boxes):
         else:
             height = tf.reshape(images.row_lengths(), (-1, 1))
             width = tf.reshape(tf.reduce_max(images.row_lengths(axis=2), 1), (-1, 1))
+            if isinstance(boxes, tf.RaggedTensor):
+                height = tf.expand_dims(height, axis=-1)
```

Comment:
@bhack can you add a comment explaining why this works?

Reply:
It was more then 1 month ago but if I remember correctly is that we need to add an extra dim in the ragged case as we are using high and width as multipliers in rel conversion.